File: app/src/main/java/com/example/thesisaudiobook/model/IdAudioBook.java
package com.example.thesisaudiobook.model;

public class IdAudioBook {

    private Integer idAudioBookPlayList;
    private String idAudioBook;

    public IdAudioBook() {
    }

    public IdAudioBook(Integer idAudioBookPlayList, String idAudioBook) {
        this.idAudioBookPlayList = idAudioBookPlayList;
        this.idAudioBook = idAudioBook;
    }

    public Integer getIdAudioBookPlayList() {
        return idAudioBookPlayList;
    }

    public void setIdAudioBookPlayList(Integer idAudioBookPlayList) {
        this.idAudioBookPlayList = idAudioBookPlayList;
    }

    public String getIdAudioBook() {
        return idAudioBook;
    }

    public void setIdAudioBook(String idAudioBook) {
        this.idAudioBook = idAudioBook;
    }

    @Override
    public String toString() {
        return "IdAudioBook{" +
                "idAudioBookPlayList=" + idAudioBookPlayList +
                ", idAudioBook='" + idAudioBook + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/thesisaudiobook/model/IdAudioBook.kt
package com.example.thesisaudiobook.model

class IdAudioBook {
    var idAudioBookPlayList: Int? = null
    var idAudioBook: String? = null

    constructor() {}
    constructor(idAudioBookPlayList: Int?, idAudioBook: String?) {
        this.idAudioBookPlayList = idAudioBookPlayList
        this.idAudioBook = idAudioBook
    }

    override fun toString(): String {
        return "IdAudioBook{" +
                "idAudioBookPlayList=" + idAudioBookPlayList +
                ", idAudioBook='" + idAudioBook + '\'' +
                '}'
    }
}